src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplyForm.scss

```diff
@@ -1,12 +1,15 @@
 @import '../../../common/variables';
 
-.bcs-ActivityThreadReplyForm-toggle {
+.btn-plain.bcs-ActivityThreadReplyForm-toggle {
     display: flex;
     align-items: center;
+    color: $bdl-gray-62;
+    font-weight: bold;
 
     &,
     &:hover {
         padding: $bdl-grid-unit $sidebarActivityFeedSpacingHorizontal ($bdl-grid-unit * 5);
+        font-weight: bold;
```

Comment:
Can we add `focus-visible` styles here also? Currently there's no visible cue when one navigates with keyboard in sidebar that they on Reply button.